src/font/lv_font.c

```diff
@@ -30,6 +30,12 @@
  *  STATIC VARIABLES
  **********************/
 
+/**********************
+ *  GLOBAL VARIABLES
+ **********************/
+
+const lv_font_t * lv_font_default = LV_FONT_DEFAULT;
```

Comment:
Could it change? If not:

```suggestion
const lv_font_t const * lv_font_default = LV_FONT_DEFAULT;
```

Reply:
Done